conf/messages.en

```diff
@@ -726,19 +719,17 @@ title.guinea=Guinea
 title.guinea_bissau=Guinea-Bissau (formerly Portuguese Guinea)
 title.guyana=Guyana
 title.haiti=Haiti
-title.hawaii=Hawaii
 title.heard_and_mcdonald_islands=Heard and McDonald Islands
 title.honduras=Honduras (including Swan Islands)
 title.hong_kong=Hong Kong
 title.howland_islands=Howland Islands
 title.hungary=Hungary
 title.iceland=Iceland
-title.lles_des_saintes=Iles des Saintes
 title.india=India
 title.indonesia=Indonesia
 title.iran=Iran
 title.iraq=Iraq
-title.irish_republic=Irish Republic
+title.irish_republic=Ireland
 title.israel=Israel
 title.italy=Italy (including Livigno; excluding the Municipality of Campione d’Italia)
 title.ivory_coast=Ivory Coast (Cote d’Ivoire)
```

Comment:
Just a query: the Ivory Coast row is marked in yellow in the spreadsheet, which seems to indicate only that the country codes need to be aligned (CI1 -> CI), not that the name needs to change*. However, the name listed in the spreadsheet is simply "Ivory Coast" omitting the parenthesised French equivalent. Assume it remains correct to include the latter and that the spreadsheet just hasn't listed the full name, but highlighting it in case that isn't the case. 

* only blue rows indicate a name change, presumably?

Reply:
Exactly, that is my understanding too. The country name should be changed only if they exist in the blue rows! In the case of Ivory Coast, only the country code needs to be aligned.